cmake/util/FindCUDA.cmake

```diff
@@ -41,20 +41,14 @@ macro(find_cuda use_cuda)
         ${CUDA_TOOLKIT_ROOT_DIR}/lib/x64
         ${CUDA_TOOLKIT_ROOT_DIR}/lib/Win32)
     else(MSVC)
+      # The CI seems to have trouble locating the CUDA library so I'm manually doing that.
       find_library(_CUDA_CUDA_LIBRARY cuda
         PATHS ${CUDA_TOOLKIT_ROOT_DIR}
         PATH_SUFFIXES lib lib64 targets/x86_64-linux/lib targets/x86_64-linux/lib/stubs lib64/stubs
```

Comment:
CI fails if I remove the stubs directory.  But the same Docker image passes on my machine @jermainewang